### ISSUE TITLE
Doc Request: add example on how to define a new unit called "pages"

### ISSUE THREAD

**Issue by DeflateAwning** [sharkdp/numbat#306]
I need to do some math thinking about pages-per-hour, etc. How can I define a new unit called `page`, so I can do math like: `10 pages / (50 pages/hour)`

I think that it would be great if this example could be added to this documentation page: https://numbat.dev/doc/unit-definitions.html

**Comment by DeflateAwning** [comment]
Update: it appears that this is the way to do it:

```
@aliases(pages)
unit page: Scalar

10 pages / 15 pages per hour
```

I maintain that this would be a great example to include in the docs, as it demonstrates that artibrary quantities representing physical object, etc. can be used and defined as scalars.

**Comment by eminence** [comment]
Hi, what about the documentation in the "Custom dimensions" section [here](https://numbat.dev/doc/dimension-definitions.html#custom-dimensions) ?

**Comment by DeflateAwning** [comment]
Yeah I guess it's there. Maybe a link from the Unit Definitions page would be good?

**Comment by eminence** [comment]
There already is one :)

![image](https://github.com/sharkdp/numbat/assets/402454/f051e9ee-45ac-487e-aedd-db40b4d9ba19)

**Comment by DeflateAwning** [comment]
Just opened a PR to make it slightly improve that link to what I expected it to be when I went looking in the docs. Very small change, but fixes my issue here.

**Comment by sharkdp** [comment]
> Update: it appears that this is the way to do it:
> 
> ```
> @aliases(pages)
> unit page: Scalar
> ```

For the record: That is *not* the way to do it. In fact, it's a bug that this is even possible (I reported it as https://github.com/sharkdp/numbat/issues/308). What you probably want to do is to define a completely new unit with a new base dimension. This can be done by defining
```
@aliases(pages)
unit page
```
without specifying a dimension. It will create a new dimension for you (called `Page`).

**Comment by sharkdp** [comment]
I hope the changes in #380 clarify things: https://numbat.dev/doc/unit-definitions.html#ad-hoc-units
